edl.py

```diff
@@ -298,7 +298,7 @@ def run(self):
                 else:
                     print("Device is in EDL mode .. continuing.")
                     self.cdc.timeout = None
-                    sahara_info = self.sahara.info()
```

Comment:
if i execute this code, the `info()` method in `sahara.py` gets called... issue is there are kind of two of them. the one that i renamed and one for logging (`info("my logging msg with log level info")`) hence i renamed it